docs/scheduler.md

```diff
@@ -0,0 +1,60 @@
+# Edge Scheduler
+
+This is a component in an architecture not yet shared in detail. It is
+an early milestone in edge multicluster, achieving partial
+functionality and implemented in a simple crude approach that is
+layered on top of transparent multicluster. In this approach the
+edge-mc layer maintains a kcp workspace, called a mailbox, per
+selected edge destination per EdgePlacement object. Then TMC syncs
+between the mailbox workspaces and the edge destinations.
```

Comment:
Does this mean there is a mailbox workspace for each edge destination and edge placement object pair?  Or there is a mailbox workspace for each edge destination and then I can have multiple edge placement objects targeting that edge destination?